app/views/home/dashboard.scala.html

```diff
@@ -149,91 +102,88 @@ <h2 class="heading-medium">2. Take the online tests</h2>
                 </div>
             </li>
 
-            <li class="third-step @setVisibility(dashboardPage.thirdStepVisibility)">
+            <li class="third-step @dashboardPage.isThirdStepVisible">
```

Comment:
Good idea but I rather like to keep it for a separate story. Also because I've already merge and fix conflict on this twice. Do we have a way to keep track in Jira tech debts or improvements?